### ISSUE TITLE
how is dataset versioning handled in this scheme?

### ISSUE THREAD

**Issue by titaniumbones** [qri-io/archivers#1]
Does the Metablocks spec also manage assertions that the present object is a version of some other object? If so, is there as yet any indication of how this would be done? If not, would there be some other mechanism for this?

**Comment by b5** [comment]
Yep. Needs clarification, but the "subject" of a metablock chain can be either a static content address, or an IPNS entry, which is a pointer to an object history.
